inc/wp-components/classes/class-head.php

```diff
@@ -22,12 +26,58 @@ class Head extends Component {
 	 */
 	public $name = 'head';
 
+	/**
+	 * Define default children.
+	 *
+	 * @return array Default children.
+	 */
+	public function default_children() : array {
+		return [
+			( new Component() )
+				->set_name( 'title' )
+				->set_children( [ get_bloginfo( 'name' ) ] ),
```

Comment:
Should this be like `->set_config( 'content', get_bloginfo( 'name' ) ),` or something? Maybe I'm misunderstanding though

Reply:
This is how Max originally built it, I can't really explain. I'd love to dig in more and figure it out, but for now it works.